pages/about.tsx

```diff
@@ -78,7 +78,7 @@ export const About: NextPage = () => {
         <Container maxW="container.xl">
           <SimpleGrid columns={[1, 1, 2]} spacing={10}>
             <Heading color="white">
-              We're part of the UVic Engineering Student Society
+              We're part of the UVic Engineering and Computer Science Students' Society
             </Heading>
             <Text color="white">
               We get funding from the UVic Engineering Student Society to
```

Comment:
This also using the previous ECSS name

Reply:
@AmandaGAnderson According to [ECSS's](https://www.uvicecss.ca/) website, it's _UVic Engineering & Computer Science Students’ Society_. Should I change it back?

Reply:
No? I was pointing out line 84 in the file says "UVic Engineering Student Society" and should also be updated if everything else referencing the ECSS is udpated

Reply:
I updated the name change in the latest commit for this pull request. Let me know if everything looks good!